### PR TITLE
doc: improve doc for lazy consensus process

### PR DESCRIPTION
**Pull request by mhdawson**
Refs: https://github.com/openjs-foundation/cross-project-council/issues/1241

### CODE REVIEW

File: governance/GOVERNANCE.md
Comment (ljharb) [inline governance/GOVERNANCE.md:64]: How is “appropriate” determined?

**Review by ljharb — APPROVED** [review]

File: governance/GOVERNANCE.md
Comment (marco-ippolito) [reply governance/GOVERNANCE.md:64]: I think we should define a number like X weeks, and if it's too much or too little we can update it

Comment (mcollina) [reply governance/GOVERNANCE.md:64]: I prefer to keep it loose here, and mention something like "after a CPC public meeting".

Comment (mhdawson) [reply governance/GOVERNANCE.md:64]: I had thought about it and different times may apply depending on the group/topic which is why I left it out I think specific times can be defined for specific cases, for example approving a new project might have a longer time than something else.

Comment (ljharb) [reply governance/GOVERNANCE.md:64]: I'm totally fine to keep it loose in the actual document, to be clear, which is why i stamped it - i was trying to get an understanding of what others' intentions/expectations here are.

File: governance/GOVERNANCE.md
Comment (tobie) [inline governance/GOVERNANCE.md:74]: > "A group voting member may choose to participate in any vote
through abstention."

I'm not exactly sure what this means.

File: governance/GOVERNANCE.md
Comment (tobie) [reply governance/GOVERNANCE.md:64]: +1 to keeping the definition non-specific here. We could indicate that the timing is context-specific, like so:

```suggestion
consensus and waiting for an appropriate amount of time given the nature of the decision for anyone to object.
```

**Review by tobie — COMMENTED** [review]
Should we modify [the charter](https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making) accordingly?

**Review by tobie — APPROVED** [review]

File: governance/GOVERNANCE.md
Comment (ljharb) [reply governance/GOVERNANCE.md:74]: To me it means that you can participate without saying yes or no, by abstaining.

**Review by tobie — CHANGES_REQUESTED** [review]
Thanks a lot for getting this started, @mhdawson.

Giving this another thought, I think we shouldn't duplicate the charter's content in the `governance.md` at all, even as a first step. `governance.md` should only reference the decision-making and voting policies of the charter.

I suggest we:

1. change this PR so that `governance.md` only references the decision-making and voting policies of the charter, and
2. bring your suggested changes directly into the charter itself, either in this PR or in a new, separate PR (whatever you feel as more convenient).

File: governance/GOVERNANCE.md
Comment (tobie) [reply governance/GOVERNANCE.md:74]: So that's the language that's also in the charter. It's either too explicit or not enough. For example, 

- is the vote of present members or all members?
- is the vote async or not?
- if the vote is of members present, do we need a quorum to have a vote?
- do votes need to be announced in advance?
- is simple majority "more than 50%" or "the answer with a bigger score"?
- if majority is "more than 50%" are abstentions counted?
- if so, what prevents reframing the question as its negative to flip the result of the vote, e.g. 4 people vote for prop `x`, 3 against, and 3 abstain. The vote fails with only 40% of the votes. Now organize a vote for `!x`. You'll get the 4 people that were for `x` vote against `!x`, the 3 that were against `x` now vote for `!x`, and the same 3 as before will abstain. So now we have only 30% for `!x` which fails, an do so `x` passes.
- can anyone request a vote at anytime? If so, that goes against the idea of being consensus-driven. If not, the language needs to be mores specific about when it is OK to ask for a vote.

I don't want us to spend too much time discussing voting as the case never really showed up, so my suggestion would be to acknowledge that voting should be a last resort solution, costly in social capital, painful to organize, hard to weaponize, and favoring the status quo.

I'd suggest the following language:

> Voting should only be done as a last resort. If a proposal to modify existing policies and processes (or add new ones) cannot reach consensus after a reasonable period of discussion, a voting member can call for objections to be overruled through an asynchronous vote. If this call is seconded by another voting member, an asynchronous vote must be organized. For the objections to be overruled (and thus the proposal to pass), at least 2/3 of all voting members must vote in favor. This is designed to make it difficult to change existing policies and processes (or to add new ones) without consensus, but prevent someone from completely blocking progress through objections.

WDYT?

**Comment by tobie** [comment]
Opened a PR against this PR here: https://github.com/mhdawson/bootstrap/pull/1

**Review by marco-ippolito — APPROVED** [review]

File: governance/GOVERNANCE.md
Comment (UlisesGascon) [reply governance/GOVERNANCE.md:74]: > WDYT?

I think the suggestion is a great addition.

**Comment by mhdawson** [comment]
@tobie landed your PR.

**Review by tobie — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]

**Review by wesleytodd — APPROVED** [review]

**Review by joesepi — APPROVED** [review]

**Comment by mhdawson** [comment]
We discussed in the CPC working meeting today. The main point of discussion was the requirement for 2/3 which is different than what was previously documented. None of the 9 people in the meeting were concerned with that. @openjs-foundation/cpc if you agree please add your +1 to the PR or make suggestions. We agreed to make one more checkin with the CPC before landing the PR based on no objections/approvals.

**Comment by tobie** [comment]
Heads-up that while @mhdawson's original PR didn't touch the charter this final version does, and so this needs Board approval.

**Review by anonrig — APPROVED** [review]

File: governance/GOVERNANCE.md
Comment (PaulaPaul) [reply governance/GOVERNANCE.md]: I also think this is a great addition/clarification of the consensus process for decision making. 
It's a nit and not needed to land these changes, but depending on who is looking at the governance doc and decision making processes, it could help to explicitly state that these processes are applicable to anything in scope for the CPC (defined in the README.md). Nice work!

Comment (tobie) [reply governance/GOVERNANCE.md]: Technically, the scope of the CPC is defined in the charter itself.

**Comment by tobie** [comment]
We just got board approval in today's board meeting.

**Pull request merged**
